specification/hardwaresecuritymodules/HardwareSecurityModules.Management/CloudHsmCluster.tsp

```diff
@@ -0,0 +1,240 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.HardwareSecurityModules;
+/**
+ * Resource information with extended details.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property"
+model CloudHsmCluster
+  is Azure.ResourceManager.TrackedResource<CloudHsmClusterProperties> {
```

Comment:
Why is the suppression over the entire model? It'll suppress this issue over every new property added. Should we be defining this in a different way?